ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/Genesis/Tests/Uniform.hs

```diff
@@ -207,10 +207,10 @@ prop_leashingAttackStalling = QC.expectFailure <$> do
 prop_leashingAttackTimeLimited :: QC.Gen QC.Property
 prop_leashingAttackTimeLimited = QC.expectFailure <$> do
   genesisTest <- genChains (QC.choose (1, 4))
-  schedulePoints <- genTimeLimitedSchedule genesisTest
+  schedule <- fromSchedulePoints <$> genTimeLimitedSchedule genesisTest
   pure $
-    runGenesisTest' schedulerConfig genesisTest (fromSchedulePoints schedulePoints) $
-    makeProperty genesisTest schedulePoints
+    runGenesisTest' schedulerConfig genesisTest schedule $
+    theProperty genesisTest schedule
```

Comment:
I think I need an account of why it is better to have properties using `PointSchedule`. Maybe in the form of a comment about `forAllGenesisTest`.

Reply:
After reviewing #856, I suppose this depends on the discussion there.